From 8b9eead90a20225e56a0b277f742a8f07015e54a Mon Sep 17 00:00:00 2001
From: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Date: Thu, 12 Jan 2023 12:56:46 +0200
Subject: [PATCH 32/63] devicetree/bindings: Document STM compatible

The compatible string "nxp,s32cc-stm-ts" will be used by the
STM instances that assume the counting will be started on the
driver's probe.

Issue: ALB-9261
Upstream-Status: Pending 

Signed-off-by: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 .../devicetree/bindings/clock/nxp-s32cc-stm-global.yaml      | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/Documentation/devicetree/bindings/clock/nxp-s32cc-stm-global.yaml b/Documentation/devicetree/bindings/clock/nxp-s32cc-stm-global.yaml
index 72b29a148b79..829221ef9ed0 100644
--- a/Documentation/devicetree/bindings/clock/nxp-s32cc-stm-global.yaml
+++ b/Documentation/devicetree/bindings/clock/nxp-s32cc-stm-global.yaml
@@ -17,6 +17,7 @@ properties:
     oneOf:
       - enum:
           - nxp,s32cc-stm-global
+          - nxp,s32cc-stm-ts
 
   reg:
     maxItems: 1
@@ -48,9 +49,9 @@ unevaluatedProperties: false
 
 examples:
   - |
-    #include <dt-bindings/clock/s32gen1-scmi-clock.h>
+    #include <dt-bindings/clock/s32cc-scmi-clock.h>
 
-    stm0: stm@4000D000{
+    stm0: stm@4000d000{
         compatible = "nxp,s32cc-stm-global";
         reg = <0x4000D000 0x1000>;
         interrupts= <0 24 4>;
-- 
2.25.1

